Sources/Tests/EndToEnd/Test_EndToEnd.c

```diff
@@ -72,7 +72,6 @@ static void ReadTemplate(const char *expectedFileName, Template *expectedTemplat
     assert(ret != EOF);
 }
 
-
```

Comment:
@david-connell can explain it better than I can, but the Unity stuff adds extra stuff into the malloc so it can track it, if we free with the normal free, rather than the Unity free, then this extra stuff is leaked.